pkg/backends/websockets.go

```diff
@@ -136,28 +132,25 @@ func newWebsocketSession(conn *websocket.Conn) *WebsocketSession {
 // Send sends data over the session
 func (ns *WebsocketSession) Send(data []byte) error {
 	buf := ns.framer.SendData(data)
-	n, err := ns.conn.Write(buf)
+	err := ns.conn.WriteMessage(websocket.BinaryMessage, buf)
```

Comment:
Does `WriteMessage()` take a whole message, which will then be presented on the other side via `ReadMessage()`?  If so then we don't actually need `framer` at all here.

Reply:
yes it takes whole message, so I removed framer.